libcxx/include/__config

```diff
@@ -1023,6 +1023,13 @@ __sanitizer_verify_double_ended_contiguous_container(const void*, const void*, c
 #    define _LIBCPP_HAS_NO_RTTI
 #  endif
 
+// Same as _LIBCXXABI_DTOR_FUNC in libcxxabi, we need this when working with ABI in exceptions code
+#  if defined(_WIN32)
+#    define _LIBCXX_DTOR_FUNC __thiscall
+#  else
+#    define _LIBCXX_DTOR_FUNC
+#  endif
+
```

Comment:
According to https://learn.microsoft.com/en-us/cpp/cpp/thiscall?view=msvc-170, it doesn't seem like this should be needed.

Reply:
MinGW refused to build in CI without it, but yes, lambda seems to fix the issue. Removed